### PR TITLE
add spinner

### PR DESCRIPTION
**Pull request by Hopsaheysa**
added spinner to DetailVC until image is loaded

### CODE REVIEW

**Comment by kober32** [comment]
Can you set me as a reviewer so I can add comments to the changes?

File: SpaceXLaunches.xcodeproj/xcuserdata/marek.xcuserdatad/xcdebugger/Breakpoints_v2.xcbkptlist [hunk 1]
```diff
@@ -36,54 +36,6 @@
             landmarkType = "7">
          </BreakpointContent>
       </BreakpointProxy>
-      <BreakpointProxy
```
Comment (kober32) [inline SpaceXLaunches.xcodeproj/xcuserdata/marek.xcuserdatad/xcdebugger/Breakpoints_v2.xcbkptlist:39]: The whole `xcuserdata` folder should be in the `.gitignore`

**Comment by Hopsaheysa** [comment]
> Can you set me as a reviewer so I can add comments to the changes?

please accept my invite to be a collaborator so I can add you as a reviewer

**Review by kober32 — COMMENTED** [review]
I added some comments. I would improve the "forever loading" state in the detail view at least.

I'm not marking it as request changes/approved as this is your project, so it's up to you marge it later.

**Review by kober32 — APPROVED** [review]
👍🏻 Nice changes

**Pull request merged**